docs/examples/ko/controls/DeviceOrientationControls.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="ko">
```

Comment:
Shouldn't this be `kr`, see #20872? I'm not sure what shortcut is right...

Reply:
Isn't `kr` the country code? I think the language code `ko` is correct.

Reply:
@hareha What do you think?

Reply:
I've checked the ISO standard and indeed it is `ko`!

https://www.loc.gov/standards/iso639-2/php/langcodes_name.php?iso_639_1=ko